apis/Google.Cloud.Spanner.Data/docs/history.md

```diff
@@ -1,5 +1,9 @@
 # Version history
 
+# Version 3.4.0, released 2020-12-14
+
+- [Commit 9fb3e43](https://github.com/googleapis/google-cloud-dotnet/commit/9fb3e43): feat: Database Admin: Adds UpdateDatabaseDdlMetadata.Throttled to indicate when a
```

Comment:
nit: It's probably the commit that's cut as well, but this here is incomplete "to indicate when a..."

Reply:
Whoops - thanks, will fix.

Reply:
(It's not the commit that's cut - it was me.)